lib/ssl/src/ssl_handshake.erl

```diff
@@ -1095,7 +1100,8 @@ add_tls12_extensions(_Version,
       next_protocol_negotiation =>
           encode_client_protocol_negotiation(NextProtocolSelector,
                                              Renegotiation),
-      sni => sni(ServerNameIndication)
+      sni => sni(ServerNameIndication),
+      mfl => mfl(MaxFragmentLength)
```

Comment:
We think this belong to the common extensions as it can be present in all TLS versions.

Reply:
SNI is also applicable to other TLS versions but resides in add_tls12_extensions which is why I put MFL here.
Please confirm you want MFL in the common extensions.